celery/worker/consumer/consumer.py

```diff
@@ -505,13 +505,14 @@ def _error_handler(exc, interval, next_step=CONNECTION_RETRY_STEP):
             # to determine whether connection retries are disabled.
             retry_disabled = not self.app.conf.broker_connection_retry
 
-            warnings.warn(
-                CPendingDeprecationWarning(
-                    f"The broker_connection_retry configuration setting will no longer determine\n"
-                    f"whether broker connection retries are made during startup in Celery 6.0 and above.\n"
-                    f"If you wish to retain the existing behavior for retrying connections on startup,\n"
-                    f"you should set broker_connection_retry_on_startup to {self.app.conf.broker_connection_retry}.")
-            )
+            if retry_disabled:
+                warnings.warn(
+                    CPendingDeprecationWarning(
+                        "The broker_connection_retry configuration setting will no longer determine\n"
```

Comment:
should we keep using the f"string" here as before?

Reply:
You mean with the `{self.app.conf.broker_connection_retry}`? I don't think so, given that I also changed "retain the existing behavior" to "refrain from retrying connections".
Actually, I copied the warning in the [doc](https://docs.celeryq.dev/en/stable/userguide/configuration.html#broker-connection-retry), I think having the exact same words here and there make a lot of sense

(Or if you were just talking about the `f` before the `"`, the linter complained about it being useless given that there's no variables inside anymore)